src/servers/ZoneServer2016/entities/zombie.ts

```diff
@@ -37,34 +36,4 @@ export class Zombie extends Npc {
     );
     this.materialType = MaterialTypes.ZOMBIE;
   }
-
-  OnInteractionString(server: ZoneServer2016, client: ZoneClient2016) {
```

Comment:
it wasn't used ?

Reply:
I tried to use the Zombie OnInteractionString, but it wouldn't work with my Npc OnInteractionString since zombie extends from Npc, so I just combined them all temporarily in the npc class, once each individual animal class is setup, this can be revised to each class having its own OnInteractionString and OnPlayerSelect.